jerry-core/debugger/debugger-ws.h

```diff
@@ -64,6 +64,23 @@ typedef struct
 } jerry_debugger_send_header_t;
 
 /**
+ * Incoming message: next message of string data.
+ */
+typedef struct
+{
+  uint8_t type; /**< type of the message */
+} jerry_debugger_receive_uint8_data_part_t;
+
+/**
+ * String data for eval and client source.
```

Comment:
Byte data for evaluating expressions and receiving client source.

(Note: in case of client source we receive two strings, so String data is not even correct now).